Swift/Tests/CBLTestHelper.m

```diff
@@ -27,4 +27,18 @@ + (void) allowExceptionIn: (void (^)(void))block {
     --gC4ExpectExceptions;
 }
 
++ (BOOL) catchException: (void(^)(void))tryBlock error: (__autoreleasing NSError **)error {
+    @try {
+        ++gC4ExpectExceptions;
+        tryBlock();
+        --gC4ExpectExceptions;
+        return YES;
+    }
+    @catch (NSException *exception) {
+        *error = [[NSError alloc] initWithDomain: exception.name code: 0
```

Comment:
Can it be *error = exception?

Reply:
Do you mean, `*error = exception`?

It's not possible, NSError and NSException type mismatch?